app/src/src/screens/menu/ProfileScreen.jsx

```diff
@@ -34,32 +35,46 @@ class ProfileScreen extends Component {
     }
   };
 
+  componentWillUnmount() {
+    this.setState = (_state, _callback) => {
+      return;
+    };
+  }
+
```

Comment:
Curious, what does this do? Does this prevent an API call from updating the state of an unmounted component?

Reply:
yup I think so, it returns null so when someone moves from the edit profile page to the menu page (or even sometimes the edit profile page to the profile page, or just in between these 3 pages), then there was an error before about how it can't perform an update on an unmounted component. So it makes it null instead to clean it up.

Reply:
This is really cool, I think we'll try implement this in other places where this error occurs.